sci-libs/primegen/primegen-0.97.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="Fast C/C++ prime number generator library"
+DESCRIPTION="primegen is a small, fast library to generate prime numbers \
+in order. It generates the 50847534 primes up to 1000000000 in just 8 \
+seconds on a Pentium II-350; it prints them in decimal in just 35 seconds. \
+primegen can generate primes up to 1000000000000000, although it is not \
+optimized for primes past 32 bits. It uses the Sieve of Atkin instead of the \
+traditional Sieve of Eratosthenes."
+HOMEPAGE="https://cr.yp.to/primegen.html"
+COPYRIGHT="1999, D. J. Bernstein"
+LICENSE="Public Domain"
+REVISION="1"
+SOURCE_URI="https://cr.yp.to/primegen/primegen-$portVersion.tar.gz"
+CHECKSUM_SHA256="54285baf8eed9e421ff2220a2112d38cfb20c1ebef6014ef3f0004c22c95f40d"
+PATCHES="primegen-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	primegen = $portVersion
+	cmd:primegaps = $portVersion
+	cmd:primes = $portVersion
+	cmd:primespeed = $portVersion
+	lib:libprimegen
+	"
+REQUIRES="
+	haiku
+	"
+
+PROVIDES_devel="
+	primegen_devel = $portVersion
+	devel:libprimegen
+	"
+REQUIRES_devel="
+	primegen == $portVersion base
+	"
+
+BUILD_PREREQUIRES="
+	cmd:make
+	cmd:gcc
+	cmd:nroff
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
+BUILD()
+{
+	make
+}
+
+INSTALL()
+{
+	install -d $binDir $docDir $manDir/man1 $manDir/man3
+	install -d $includeDir/primegen $developLibDir $libDir
+	install -t $binDir primegaps primes primespeed
+	install -t $docDir README
```

Comment:
I saw that the README mentions another interesting file, BLURB.
I think it would be nice to also add it to the docDir.
Don't worry, I won't be picky this time ;-)))) I'll be happy with any of these:
```
	install -t $docDir README BLURB
```
or
```
	install -t $docDir BLURB README
```

BTW, if/when you squash your commits, the most used style for commit messages (in situations like this one) is:
```
primegen: add initial recipe.
```

Regarding the squash, it's nice if you do it yourself, but that's not required, since GitHub has a "Squash and merge" button which works fine.